lib/src/widgets/board.dart

```diff
@@ -729,13 +757,18 @@ class _BoardState extends State<Chessboard> {
       return;
     }
 
+    final square = widget.offsetSquare(details.localPosition);
+
+    if (square != null && square == _currentPointerDownSquare) {
```

Comment:
I think we still want to have this block after the
```dart
    if (_currentPointerDownEvent == null ||
        _currentPointerDownEvent!.pointer != details.pointer) {
      return;
    }
```

logic.

Reply:
No that doesn't work, even with the other listened always enabled, `_currentPointerDownEvent` will be `null` if the board is non-interactive